src/modules/add.js

```diff
@@ -1,10 +1,10 @@
 let ScoresArray = [];
 
-export function addtoLocalStorage() {
+export const addtoLocalStorage = () => {
   localStorage.setItem('data', JSON.stringify(ScoresArray));
-}
+};
 
-export function displayScores() {
+export const displayScores = () => {
   const listItems = document.querySelector('.list-items');
   listItems.innerHTML = '';
 
```

Comment:
- [ ] I have to approve due to consistency, however, you have not met the requirements for this milestone. You have to send the scores via the API using the `POST` method and then retrieve the data using the `GET` method via the API.
Kindly refer to this [resource](https://www.topcoder.com/thrive/articles/fetch-api-javascript-how-to-make-get-and-post-requests) for more information on how to do so

### Fetching Data Example
```
async function fetchData() {
  try {
    const response = await fetch('https://api.example.com/data'); // Replace with your API endpoint URL
    if (!response.ok) {
      throw new Error('Network response was not ok.');
    }
    const data = await response.json();
    return data;
  } catch (error) {
    console.error('Error fetching data:', error);
    return null;
  }
}

// Usage
async function getDataFromApi() {
  const data = await fetchData();
  if (data) {
    console.log('Fetched data:', data);
    // Process the data as needed
  }
}
getDataFromApi();
```

### Posting Data Example
```
async function postData() {
  const dataToSend = { name: 'John', age: 30 };
  try {
    const response = await fetch('https://api.example.com/data', {
      method: 'POST',
      headers: {
        'Content-Type': 'application/json',
      },
      body: JSON.stringify(dataToSend),
    });
    if (!response.ok) {
      throw new Error('Network response was not ok.');
    }
    const result = await response.json();
    return result;
  } catch (error) {
    console.error('Error posting data:', error);
    return null;
  }
}

// Usage
async function postDataToApi() {
  const result = await postData();
  if (result) {
    console.log('Posted data successfully:', result);
    // Process the response as needed
  }
}
postDataToApi();
```